### ISSUE TITLE
The expanded list of the AutoComplete is not announced by screen readers

### ISSUE THREAD

**Issue by IvanDanchev** [telerik/kendo-ui-core#6223]
### Reproduction of the problem
Reproducible in the [demos](https://demos.telerik.com/kendo-ui/autocomplete/keyboard-navigation).
1. Focus the AutoComplete and type in "a"

### Current behavior
The list of items shows up, but the reader does not announce this. 

### Expected/desired behavior
The reader should announce that the list is expanded, as it does for the ComboBox. To achieve that the role of the widgets should be set to "combobox"

### Environment

* **Kendo UI version:** 2020.3.1118
* **jQuery version:** x.y
* **Browser:** [all]

**Comment by Juveniel** [comment]
This item will be reviewed as part of https://github.com/telerik/kendo/issues/11687

**Comment by veselints** [comment]
Implemented with:
https://github.com/telerik/kendo/issues/12164

**Comment by Dimitar-Goshev** [comment]
The reported AutoComplete behavior is expected. The AutoComplete has no focused dropdown item when filtering, thus list items are not read by screen readers. The ComboBox has focused item in this scenario, thus list items are read. To focus the first list item in AutoComplete, set its [highlightFirst](https://docs.telerik.com/kendo-ui/api/javascript/ui/autocomplete/configuration/highlightfirst) property to true - [Dojo](https://dojo.telerik.com/ERaQITEy) .
